src/shared/components/query/studyList/StudyList.tsx

```diff
@@ -445,6 +448,21 @@ export default class StudyList extends QueryStoreComponent<
             );
         }
     };
+
+    private renderReferenceGenomeBadge(study: CancerStudy) {
+        return study.referenceGenome ? (
+            <span
+                className={`badge badge-pill badge-secondary ${styles.badgeReferenceGenome}`}
+                style={{
+                    backgroundColor:
+                        referenceGenomeBadgeColors[study.referenceGenome] ||
+                        'grey',
```

Comment:
we should try to avoid using arbitrary color names like this.   Look up variables.scss in our project and you will find a list of colors.  Perhaps $mediumGray would be good since it's used for the sample pills on patient page.   There are a couple ways to access these.  One would be to use the variable in baedReferenceGenome class.  Then here, you only set backroundColor if one resolves from rferenceGenomeBadgeColors.    There is also a way of exporting variables from the scss file.  We've never done this, but it's 
```
:export {
  primaryColor: $primary-color;
  backgroundColor: $background-color;
  padding: $padding;
}
```
And then you can import them supposedly.

Reply:
the former strategy is probably simplest?  except that you then have conditionally apply the backgroungColor in the style={}.  Just a little more code.

Reply:
Exporting out of a scss file didn't work for me, I added grey to the study other theme colors:
https://github.com/cBioPortal/cbioportal-frontend/pull/4407/commits/765125bb1ba294e20e059cfe213bce3d7712325b#diff-02e4860ff55191a861f21c782c3685de5c6db94180bf7daa276ed1dc14ab1765R267